reactComponents/src/demo.tsx

```diff
@@ -0,0 +1,245 @@
+      buttonColor: '#7A808A',      
+      buttonBorderRadius: '4px',      
+      buttonHoverBackColor: 'black'
+  },
+  content: {
+      padding: '14px',
+      background: '#00070E',
+      borderRadius: '4px',    
+      width: 'auto',
+      height: '300px',
+      border: '1px solid grey',   
+      color: '#FFF',
+      display: 'block',   
+      borderColor: '#474F5C',  
+      borderStyle: 'solid',  
+      borderWidth: '1px',  
+      fontSize: '15px',  
+      fontFamily: "'Fira Code', monospace'" 
+  }
+}
+
+const customLoading ={
+  loadingTitle: 'Searching...',
+  notFoundTitle: 'No results found',
+  color: 'white',
+  fontSize: '12px'
+}
+
+const customChip = {
+  fontSize: '8px',
+  fontWeight: '500',
+  borderRadius: '4px',
+  backgroundColor: '#232B35',
+  border: 'solid 2px #232B35', 
+  padding: '7px 5px', 
+  margin: '5px', 
+  defaultColor: '#B4BBC7', 
+  width: '122px', 
+  height: 'auto', 
+  textAlign: 'left',
+  textTransform: 'uppercase', 
+  gridTemplateColumns: '25px 85px 10px',
+  justifySelf: 'end',     
+  checkedBorderColor: '#474F5C',  
+  checkedColor: 'white', 
+  checkedBackgroundColor: '#474F5C'
+}
+
+const customAllChip = {
+  fontSize: '9px',
+  fontWeight: '100',
+  borderRadius: '4px',
+  backgroundColor:'#474F5C',
+  border: '0',
+  padding: '4px 10px',
+  margin: '0',
+  defaultColor: '#7A808A',   
+  width: 'auto',
+  height: 'auto',
+  textAlign: 'center' ,
+  textTransform: 'inherit',
+  gridTemplateColumns:  '40px',
+  justifySelf: 'center'
+}
+
+const customTokenDetail = {
+  list: {
+      container: {
+          display: 'grid',
+          alignItems: 'center',            
+          padding: '5px 0',
+          background: '#00070E',
+          borderBottom: '1px solid #474F5C',
+          gridTemplateColumns: '18% 1% 18% 5% 6% 37% 10%'
+      },
+
+      token: {
+          color: '#B4BBC7',
+          fontSize: '12px',
+          fontWeight: '600'
+      },
+
+      pair: {
+          color: '#B4BBC7',
+          fontSize: '8px'
+      },
+
+      detail: {
+          padding: '3px'
+      },
+
+      button: {
+          borderColor: '#474F5C',
+          backgroundColor: '#474F5C', 
+          color: '#7A808A',
+          borderRadius: '4px',
+          fontSize: '10px',
+          padding: '3px',
+          hoverBackColor: '#232C38'
+      }
+  },
+  details: {
+      content: {
+          display: 'block',
+          alignItems: 'center',
+          padding:'5px 13px',
+          margin: '5px 0',
+          background: '#474F5C',
+          borderBottom: '1px solid #474F5C',
+          borderRadius: '4px',
+          fontSize: '10px',
+          gridTemplateColumns: '52% 48% 1%'
+      },
+
+      token: {
+          fontSize: '12px',
+          fontWeight: '600'
+      },
+
+      address: {
+          fontSize: '10px'
+      },
+
+      detail: {
+          fontSize: '10px'
+      }
+
+
+  }
+}
+const rootElement = document.getElementById("root");
```

Comment:
demo - this looks cool. Is this a way to test the component by chance?

Reply:
assuming we want to test other components, should this be moved to a directory/renamed so we have one demo like this for each react component that we build later?

Reply:
Yes, but I add the rootElement into the index.tsx locally while I test.